docs/.vuepress/public/assets/migrations/scripts/migrate-3.4.0.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const fs = require('fs');
+
+const ONE_RELATIONS = ['oneToOne', 'manyToOne', 'oneWay'];
+
+const getStrapiIndexPath = projectPath => {
+  if (!projectPath) {
+    throw new Error(`
+-> Path to strapi project is missing.
+-> Usage: node migrate-3.4.0.js [path]`);
+  }
+
+  const strapiIndexPath = path.resolve(projectPath, 'node_modules', 'strapi', 'lib', 'index.js');
```

Comment:
why not do a simple require('strapi') ?

Reply:
So the script can be run from anywhere

Reply:
you can use the require(require.resolve('strapi', projectPath))